src/components/application_manager/src/rpc_service_impl.cc

```diff
@@ -631,15 +631,15 @@ bool RPCServiceImpl::ConvertSOtoMessage(
     case 0: {
       if (protocol_version == 1) {
         if (!formatters::CFormatterJsonSDLRPCv1::toString(
-                message, output_string, remove_unknown_parameters)) {
+                message, output_string, !allow_unknown_parameters)) {
```

Comment:
This is completely optional, but as you mentioned we should avoid negative statements for boolean values. `remove_unknown_params` is still used for the `applySchema`, `unapplySchema` and `toString` functions.

Reply:
We can't do this for any functions that already had the parameter before the release, since that would fundamentally change the meaning of the parameter, meaning it would be a breaking change.
There are a few other places where the parameter was added in this release, but I'm not too worried about those, especially since most of those are on the same layer as the functions which already had the parameter. In this sense, the naming is at least consistent within the RPC formatter/SmartObject schema layer.
I think we can hold off changing the other instances of this until the next major release, where we can make everything consistent.